.env

```diff
@@ -0,0 +1,5 @@
+COUCHDB_USER=user
```

Comment:
is this file needed? I saw you have default values in the code.

Reply:
You're right, I've removed the .env file, see my commit below.

Reply:
I feel like the config is a little clunky at the minute, particularly from a user perspective. If you decide to merge this (and I have time) I might open an issue and we could discuss how to clean it up a little.

Reply:
The get database from environment variables are designed for `docker run --link` which will set the `MYSQL_NAME`, `MYSQL_PORT_3306_TCP_ADDR` stuff for you.

https://github.com/binux/pyspider/pull/925/files/afb0afa970c78e299dcdc51daaed606df8427491..da12587a80925217fea8074a5839e1c6607595cb#diff-935b7ec1b99ab6d049882d35304d0d7eR104

As it's a legacy feature, there is no point to keep using it, that you don't need to set those wired environment variables.

Reply:
As docker compose now support relative path. You could just point it to config_example.json and set connection to database there.